### PR TITLE
8. Add tabs

### PR DESCRIPTION
**Pull request by nikita-resh**
image
image

### CODE REVIEW

File: src/states/Tabs/Tabs.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        >
+          <LinkTab to="/cards" label="All cards" className="black" value={0} />
+          {cards.map((item, index) => (
+            <LinkTab
+              to={`/cards/${index + 1}`}
+              label={`Card ${index + 1}`}
+              className="black"
+              key={item.id}
+              value={index + 1}
+            />
```
Comment (ArtsiomLysianok) [inline src/states/Tabs/Tabs.js:33]: Why do you use an array index ?

File: src/components/Card/Card.js [hunk 2]
```diff
@@ -49,4 +50,20 @@ const CustomCard = ({ cardTitle, cardText, editMode, setSelected, cardId }) => {
   );
 };
 
+export const ParticularCard = ({ cardList, editMode, setSelected }) => {
+  const params = useParams();
+  const id = params.id;
+  const card = cardList[id - 1];
```
Comment (ArtsiomLysianok) [inline src/components/Card/Card.js:56]: Using an array index instead of a card identifier

File: src/states/Tabs/Tabs.js (hunk 1 above)
Comment (nikita-resh) [reply src/states/Tabs/Tabs.js]: Fixed

File: src/components/Card/Card.js (hunk 2 above)
Comment (nikita-resh) [reply src/components/Card/Card.js]: Fixed.

**Review by ArtsiomLysianok — APPROVED** [review]

**Pull request merged**